30647.0.txt

```diff
@@ -0,0 +1,52 @@
+@0
+
+
+@1
+
+
+MOV UP,ACC    # Take in.a
+MOV ACC,RIGHT # Send it to the right port
+NOP           # Wait two cycles!
```

Comment:
Why is this needed? This is definitely a case where the comment should explain "why" rather than "what"